cargo-quickinstall/src/args.rs

```diff
@@ -68,3 +68,62 @@ pub fn crate_name_from_positional_args(
     }
     Ok(crate_name)
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use std::ffi::OsString;
+
+    const MOCK_CRATE_NAME: &str = "foo";
+    const MOCK_CRATE_VERSION: &str = "3.14.15";
+    const REDUNDANT_ARG: &str = "bar";
+    const INVALID_FLAG: &str = "--an-invalid-flag";
+
+    #[test]
+    fn test_options_from_env() {
+        let mock_cli_args: Vec<OsString> = vec![
+            OsString::from(MOCK_CRATE_NAME),
+            OsString::from("--version"),
+            OsString::from(MOCK_CRATE_VERSION),
+            OsString::from("--dry-run"),
+        ];
```

Comment:
(optional) `[...].iter().map(OsString::from).collect()` or something to avoid repeating OsString::from.